supplementary_materials/qwen2/chat.py

```diff
@@ -0,0 +1,124 @@
+import argparse
+from typing import List, Tuple
+from threading import Thread
+import torch
```

Comment:
there is no torch listed in requirements, possibly better to add explicitly for avoid getting model without SDPA due to too old pytorch

Reply:
yes, but we did not control version here I recommend to add with lowerbound >= 2.1